core/tx_pool_test.go

```diff
@@ -136,7 +136,7 @@ func (bc *testBlockChain) GetFeeConfigAt(parent *types.Header) (commontype.FeeCo
 }
 
 func (bc *testBlockChain) SenderCacher() *TxSenderCacher {
-	return newTxSenderCacher(1)
+	return &TxSenderCacher{}
```

Comment:
This avoids starting goroutines.
Not a very nice way to shut down this without lots of textual modifications.. So I prefer this.